packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -31,14 +31,13 @@ import type { ExpressionErrors } from "../../parser/util";
 import { PARAM } from "../../util/production-parameter";
 import { Errors } from "../../parser/error";
 
+type AccessibilityModifier = "public" | "private" | "protected";
```

Comment:
You can access `N.Accessibility` from https://github.com/babel/babel/blob/759a82abb1f2ea64c03dc9b298aa03950690fa27/packages/babel-parser/src/types.js#L661